pkgs/by-name/op/open-adventure/package.nix

```
@@ -0,0 +1,71 @@
+{
+  lib,
+  stdenv,
+  fetchFromGitLab,
+  python3Packages,
+  pkg-config,
+  libedit,
+  cppcheck,
+  coreutils,
+}:
+
+stdenv.mkDerivation (finalAttrs: {
+  pname = "open-adventure";
+  version = "1.20";
+  src = fetchFromGitLab {
+    owner = "esr";
+    repo = "open-adventure";
+    tag = finalAttrs.version;
+    hash = "sha256-xbsMz99CNLhpM6BSJVcRzxPB6tUYfPy/3Z+8BKt8b1E=";
+  };
+
+  nativeBuildInputs = [
+    python3Packages.python
+    python3Packages.pyyaml
+    pkg-config
+  ];
+
+  buildInputs = [ libedit ];
+
+  doCheck = true;
+
+  nativeCheckInputs = [
+    python3Packages.pylint
+    cppcheck
+  ];
+
+  preCheck = ''
```

```suggestion
  postPatch = ''
```
This can be merged with `postPatch`.